contrib/containerized-pbench/README

```diff
@@ -0,0 +1,8 @@
+mkdir var_lib_pbench-agent
+mkdir fiotest
+vi example.sh && chmod 755 example.sh
+podman run --rm --network host --name example-with-bash -v ./example.sh:/example.sh:z -v ./var_lib_pbench-agent:/var/lib/pbench-agent:z -v ./fiotest:/fiotest:z quay.io/pbench/pbench-agent-all-centos-8:latest /example.sh
```

Comment:
This invocation might want to map in `~/.ssh` into the container as well, if the benchmark needs to run `ssh`.

Reply:
Added.